infra/aws/terraform/prow-build-cluster/bastion.tf

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 data "aws_ami" "ubuntu" {
   most_recent = true
-  owners      = ["099720109477"] // Canonical
+  owners      = ["099720109477"] # Canonical
```

Comment:
in some places we use data sources (great!) for getting the AMI ID, and others we specify the AMI ID as a static string - any reason for the static string or can we standardize on the data source so the correct AMI is pulled based on the given Kubernetes version?